### PR TITLE
Artifact name fixes as discussed in #26.

### PR DESCRIPTION
**Pull request by lsm5**
PR #26 contained some changes for fixing artifact names and getting rid of source files in the final artifact list. This PR incorporates those changes into the main branch.

```release-note
None
```

### CODE REVIEW

**Comment by baude** [comment]
is the `stage` prefix expected?

**Comment by lsm5** [comment]
> is the `stage` prefix expected?

That's how it used to be on main branch. Should that be changed to just `podman-machine` ?

**Comment by lsm5** [comment]
@baude on aarch64, the tests seem to be failing in rootless mode https://artifacts.dev.testing-farm.io/71e92fbd-56a2-4f0e-ac85-4b5a58b03d02/ . Should I try switching to rootful ? x86_64 failures are a known infra problem that I think is being worked on.

**Comment by Luap99** [comment]
podman machine will not run as root, so we must run the verify suite rootless.

File: util.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Install deps
+dnf -y install jq dosfstools
+
```
Comment (Luap99) [inline util.sh:5]: can you drop the install here, this was added in https://github.com/containers/podman-machine-os/pull/34

**Review by Luap99 — COMMENTED** [review]
Naming changes LGTM but please drop the dnf install from the script

File: util.sh (hunk 1 above)
Comment (lsm5) [reply util.sh:5]: done

**Comment by lsm5** [comment]
@Luap99 @baude PTAL

**Review by Luap99 — APPROVED** [review]
LGTM

**Pull request merged**